sphinx/builders/epub3.py

```diff
@@ -275,7 +308,7 @@ def setup(app):
     app.add_builder(Epub3Builder)
     app.connect('builder-inited', validate_config_values)
 
-    app.add_config_value('epub_description', '', 'epub3', string_classes)
+    app.add_config_value('epub_description', 'unknown', 'epub3', string_classes)
```

Comment:
Why is this changed? I think "unknown" is not a good default setting. And users would not be noticed if this will be filled by default. I think the description should be filled by user manually.

Note: If changed, we should also update `doc/config.rst`.